bin.sh

```diff
@@ -1,32 +1,73 @@
 #!/bin/bash
 
+# Default parameters
+ALLOW_SELF_REGISTER="true"
+IPV4_RANGES=""
+IPV6_RANGES=""
+ALLOWED_SIGN_UP_EMAIL_DOMAINS=""
+
+# Parse command-line arguments for customization
+while [[ "$#" -gt 0 ]]; do
+    case $1 in
+        --disable-self-register) ALLOW_SELF_REGISTER="false" ;;
```

Comment:
maybe `DISABLE_SELF_REGISTER`?

Reply:
Options on exec commands should not be uppercase. Any reasons why you looking for uppercase?

Reply:
I'm sorry, I was confused, let me resolve this